README.md

```diff
@@ -8,23 +8,58 @@ Alpakka [![scaladex-badge][]][scaladex] [![travis-badge][]][travis] [![gitter-ba
 [gitter]:                    https://gitter.im/akka/akka
 [gitter-badge]:       https://badges.gitter.im/akka/akka.svg
 
-This project provides a home to Akka Streams connectors to various technologies, protocols or libraries.
+Systems don't come alone. In the world of microservices, cloud deployment and services, and our history of existing solutions we need to integrate. [Reactive Streams](http://www.reactive-streams.org/) give us a technology-independent tool to let these systems communicate without overwhelming each other.
+Alpakka's connectors let you stream data in and out of other technologies using [Akka's](https://doc.akka.io/docs/akka/current/stream/index.html) implementation of the Reactive Streams specification. 
+
 
 Documentation
 -------------
 
-See our [documentation](http://developer.lightbend.com/docs/alpakka/current/) and [scaladoc](http://developer.lightbend.com/docs/api/alpakka/current/akka/stream/alpakka/index.html) pages.
+The **Alpakka reference documentation** is available at [Lightbend Tech Hub](http://developer.lightbend.com/docs/alpakka/current/).
+
+The **Alpakka Kafka connector documentation** is available at [akka.io](https://doc.akka.io/docs/akka-stream-kafka/current/).
+
+To get a grip of the latest Alpakka releases check out [Alpakka releases](https://github.com/akka/alpakka/releases) and [Alpakka Kafka connector releases](https://github.com/akka/reactive-kafka/releases).
+
+
+Community
+---------
+
+You can join these groups and chats to discuss and ask Akka and Alpakka related questions:
+
+- Forums: [discuss.lightbend.com](https://discuss.lightbend.com/c/akka)
+- Chat room about *using* Akka and Alpakka: [![gitter: akka/akka](https://img.shields.io/badge/gitter%3A-akka%2Fakka-blue.svg?style=flat-square)](https://gitter.im/akka/akka)
+- Issue tracker: [![github: akka/alpakka](https://img.shields.io/badge/github%3A-issues-blue.svg?style=flat-square)](https://github.com/akka/alpakka/issues)
+
+In addition to that, you may enjoy following:
+
+- The [Akka Team Blog](https://akka.io/blog/)
+- [@akkateam](https://twitter.com/akkateam) on Twitter
+- Questions tagged [#alpakka on StackOverflow](http://stackoverflow.com/questions/tagged/alpakka)
 
-Contributions & Maintainers
----------------------------
+The Alpakka project is supported by [Lightbend](https://www.lightbend.com/) and their Alpakka team.
```

Comment:
Please note that not all modules of the project are supported, and some may be still incubating. Take note of the maturity status of each module which is documented on each of the module pages.

or something like this?

Reply:
`support` here has a double meaning. One is the support under Lightbends subscription, which currently none of the connectors in this repo have (only Reactive Kafka). And the other support is the commitment by Lightbend by forming the Alpakka team. And we work an all connectors here. We should probably use `support` only for the former. And the latter we can call `maintain`.

Reply:
Thank you for spotting this. @2m is right that I tried to use the other meaning here.